web/packages/teleterm/src/ui/Search/pickers/ActionPicker.tsx

```diff
@@ -767,8 +767,12 @@ export function AppItem(props: SearchResultItem<SearchResultApp>) {
         gap={1}
       >
         <Text typography="body1">
-          {isWebApp(app) || app.awsConsole ? (
-            <>Launch {$appName} app in the browser</>
+          {isWebApp(app) || app.awsConsole || app.samlApp ? (
+            app.samlApp ? (
```

Comment:
Did you consider extracting this to a separate function that accepts `$appName`? This way we could avoid a nested ternary and it'd be easier to see which app type maps to which copy.

It seems like most of the time, a ternary with one of the conditions nested inside within another ternary can be decomposed into a series of simpler `if`s. Maybe mustache templates were right with being logic-less after all. 😏

Reply:
Yeah, much simpler now.